platform/index.md

```diff
@@ -1,18 +1,70 @@
+
+# ServiceControl - gather all the data
+
+ServiceControl is the brain of monitoring in Particular Service Platform. It collects data on every single message flowing through your system (Audit Queue), errors (Error Queue), as well as additional information regarding sagas, endpoints heartbeats and custom checks (Control Queue). The information then is exposed to ServicePulse and ServiceInsight via HTTP API and SignalR notifications.
+
+It is important to understand, that the data is still collected even if ServiceControl is down. When it starts working again, it will process all the information that was saved in the meantime.
+
+In order to allow ServiceControl for gathering information, you need to configure your solution properly:
+
+* [enable auditing](nservicebus/operations/auditing.md) to collect data on individual messages;
+* [install plugins on your endpoints](servicecontrol/plugins.md) to monitor their health, sagas and use custom checks.
+
+By default ServiceControl stores information for 30 days and initiates purging process every minute, but you can easily customize those values using [configuration](servicecontrol/creating-config-file.md) or [command-line arguments](servicecontrol/how-purge-expired-data.md).
+
+The standard approach is to have one ServiceControl installation on a single host. ServicePulse and ServiceInsight can be installed on other machines. You need to configure ServicePulse and ServiceInsight to connect with ServiceControl. You can also make ServiceControl highly available, to avoid having a single point of failure in your monitoring system.
+
+# ServiceInsight - invaluable developer tool
+
+ServiceInsight is a WPF application with features tailored to developers needs. It allows for advanced debugging, tracking the flow of the individual message in the system, observing sagas and more. 
+
+Thanks to the rich visual content, such as flow diagrams, it is much easier to quickly spot anomalies and incorrect behaviour. At the same time, it gives access to more detailed information, such as message headers and all message metadata. For more information, including sample screens refer to [documentation](serviceinsight/getting-started-overview.md).
+
+
+# ServicePulse - production monitoring
+
+ServicePulse is a web application dedicated mainly to administrators. It gives you a clear, near real-time, high-level overview of the system. 
+
+There you will get notified when the endpoint is down or when a message fails. You can also specify your own custom checks and get alerts. The interface allows you to perform the common operations for failure recovery, such as restarting failed messages. You can also [subscribe to publicly exposed events](servicepulse/event-types.md), in order to display and handle them in a custom way.
```

Comment:
`event-types` actually refers specifically to the the SP UI. if you want to subscribe to things that is an SC concept an not an SP one. and the link should go to http://docs.particular.net/servicepulse/custom-notification-and-alerting-using-servicecontrol-events